src/plugin/plugin.ts

```diff
@@ -609,8 +609,14 @@ export interface PurchasesConfiguration {
   userDefaultsSuiteName?: string;
   /**
    * iOS-only, will be ignored for Android.
-   * Set this to FALSE to disable StoreKit2.
+   * Set this to TRUE to enable StoreKit2.
    * Default is FALSE.
```

Comment:
Hmm if the hybrids default this parameter to FALSE, in the natives, we won't know whether this was explicitly set to FALSE or not set and we should calculate the value ourselves... I think we should add a new method for the hybrids that accepts not passing `usesStoreKit2IfAvailable` that way natives could know what the developer did on the hybrids. Thoughts?

Reply:
Right, this works for now, but if we make this parameter dynamic from the backend in the future, we will have to remember to fix this in the hybrids. We can go with this for now though, I agree 👍